dk.sdu.se4.screen/src/main/java/dk/sdu/se4/screen/GameScreen.java

```diff
@@ -69,6 +71,11 @@ public void render(float deltaTime) {
         }
         // Draw the User interface
         this.game.getBatch().draw(ui,0,0);
+        // Draw the player score
+        bitmapFont.setColor(toFloatBits(151, 26, 26, 255));
+        bitmapFont.setScale(1.25f, 1.25f);
+        bitmapFont.draw(this.game.getBatch(), "42069", 95, game.getHeight() - 26);  // TODO: Implement me plz.
```

Comment:
Ser meget godt ud :) 
men hvad  mangler der her //TODO Implements me plz

Reply:
Man kan bruge entity.getPart(ScorePart) eller noget i den dur til at hente scoren fra spilleren. Så inde der hvor i skriver 42069, der skal den så vise playerens scorepart.

Reply:
lav en commonplayer modul hvor player.class ligger og brug den i GameScreen til at få scoreparten

ScorePart sp;

for(Entity e : this.mapService.getEntities(Player.class) {
     sp = e.getPart(ScorePart.class);
}

og så bruge sp.getCurrentScore() istedet for 42069.